### ISSUE TITLE
There is no description of the @export variable in the inspector

### ISSUE THREAD

**Issue by Scream034** [godotengine/godot#88703]
### Tested versions

- Reproducible in: 4.2.stable_mono, v4.3-dev3_mono

### System information

Windows 10.0.22621 - Vulkan (Forward+) - dedicated AMD Radeon RX 580 2048SP (Advanced Micro Devices, Inc.; 31.0.21910.5) - Intel(R) Xeon(R) CPU E5-2670 v3 @ 2.30GHz (24 Threads)

### Issue description

The description of the variable with @export is not displayed. It is displayed only if you have specified one such variable outside of all categories.

Displayed:
```gdscript
extends Node2D

## Description of general_property
@export var general_property: float = 666

@export_category("Some")
## Description of one with category
@export var property_some_one: int = 100
```

Not displayed:
```
extends Node2D

@export_category("Some")
## Description of one with category
@export var property_some_one: int = 100
```

### Steps to reproduce

**Open** and **view it in the editor**.

### Minimal reproduction project (MRP)

[**MRP.zip**](https://github.com/godotengine/godot/files/14384616/MRP.zip)

**Comment by ajreckof** [comment]
I can reproduce I also found that it will not show the Category created automatically this might not be related.

**Comment by dalexeev** [comment]
* Duplicate of https://github.com/godotengine/godot/issues/64432#issuecomment-1941676792.
* Fixed by #88318 for future 4.3.
* Will be cherry-picked for future 4.2.x.

Thanks for reporting the bug nonetheless!

**Comment by Scream034** [comment]
I will wait for the correction!
